src/br/com/ciandt/vnardello/service/ComentarioService.java

```diff
@@ -0,0 +1,34 @@
+package br.com.ciandt.vnardello.service;
+
+/*
+ * Metodo que retorna se a mensagem � null ou n�o
+ * Exemplo {Se o comentario for nulo, o metodo retorna falso} 
+ * return comentario
+ */
+public class ComentarioService {
+
+	private final static String COMENTARIO_REMOVIDO = new String("Coment�rio removido!");
+
+	public ComentarioService() {}
+	
+	/*
+	 * @param comentario - {Passa uma String por parametro}
+	 * 
+	 */
+	public Boolean verificarComentario(String comentario) {
+		
+		//Se o comentario for diferente de null, o retorno vai ser TRUE, senao, retorna FALSE
+		if(comentario != null) {
```

Comment:
faltou espaço após o if 
```suggestion
		if (comentario != null) {
```